README.md

```diff
@@ -1,24 +1,31 @@
-# README
+# Firefish Project - Travel app
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+This app can be used to plan for travels by building an itenerary. Firefish students and React students will team up to practice everything learned from previous classes.
 
-Things you may want to cover:
+## Google Drive Link
+https://github.com/users/ElenaCherpakova/projects/2/views/1
 
-* Ruby version
+## Front-End Repo
+https://github.com/anniethewebcoder/firefish-practicum-frontend
 
-* System dependencies
+## MVP and Stretch goals
+https://docs.google.com/document/d/11x6IiLobgSv9nSVWiJgmuIsjhN1IFt5iNeCNvN7XNz0/edit
 
-* Configuration
+## Setting Up Rails
 
-* Database creation
+https://classes.codethedream.org/course/ruby-on-rails/firefish?page=machine-setup
 
-* Database initialization
+The article above has instructions for Mac, Ubuntu, and Windows. 
 
-* How to run the test suite
+Note: We are using Ruby version 3.2.1
+Rails version is not that important
```

Comment:
I was thinking that the readme just doesn't need to mention the Rails version 😅. Not a big deal, though!